### PR TITLE
Rename CertifyPkg to PkgEqual

### PR DESCRIPTION
**Pull request by jeffmendoza**
need to rebase after #646

### CODE REVIEW

File: cmd/graphql_playground/main.go [hunk 1]
```diff
@@ -19,9 +19,12 @@ package main
 // development. Do not use in production!
 
 import (
+	_ "net/http/pprof"
```
Comment (mihaimaruseac) [inline cmd/graphql_playground/main.go:22]: Should we remove the profiling lines?

**Review by mihaimaruseac — APPROVED** [review]

File: cmd/graphql_playground/main.go (hunk 1 above)
Comment (jeffmendoza) [reply cmd/graphql_playground/main.go:22]: oops =)

File: pkg/assembler/graphql/schema/pkgEqual.graphql [hunk 2]
```diff
@@ -15,17 +15,17 @@
...
-CertifyPkg is an attestation that represents when a package objects are similar
+PkgEqual is an attestation that represents when a package objects are similar
 
 packages (subject) - list of package objects
 justification (property) - string value representing why the packages are similar
 origin (property) - where this attestation was generated from (based on which document)
 collector (property) - the GUAC collector that collected the document that generated this attestation
 """
-type CertifyPkg {
+type PkgEqual {
```
Comment (lumjjb) [inline pkg/assembler/graphql/schema/pkgEqual.graphql:28]: can we make changes to `examples/` as well?

**Review by lumjjb — APPROVED** [review]
minor comment else LGTM (needs to wait for rebase)

File: pkg/assembler/graphql/schema/pkgEqual.graphql (hunk 2 above)
Comment (jeffmendoza) [reply pkg/assembler/graphql/schema/pkgEqual.graphql:28]: Will do!

File: cmd/graphql_playground/main.go (hunk 1 above)
Comment (jeffmendoza) [reply cmd/graphql_playground/main.go]: removed

File: pkg/assembler/graphql/schema/pkgEqual.graphql (hunk 2 above)
Comment (jeffmendoza) [reply pkg/assembler/graphql/schema/pkgEqual.graphql:28]: done

**Pull request merged**